### DISCUSSION TITLE
Tasks like Confluence (or better)

### DISCUSSION THREAD

**Discussion by Arvoo-Falco** [outline/outline#3370] [General]
Hi!

We're looking into self-hosted alternatives to Confluence and I encountered your software Outline. It looks like a good alternative, but I can't seem to find anywhere if tasks are a supported functionality and how extensive or limited this is. Some features Confluence does have for tasks:

- Assign a task to a person
- Assign a due date to a task
- Create a list of all tasks that are assigned to you

Some features that Confluence doesn't have:

- Make a task recurring
- Assign a task to multiple people or make dependencies
- Synchronize tasks with an agenda

Our use case is our ISO 27001 ISMS we're running in Confluence. Part of this ISMS is making planned tasks, which are currently planned in a separate agenda and thus aren't easy to monitor from our ISMS being Confluence.

I hope someone who uses this or made this can give some more information about this!

**Comment by almereyda** [comment]
I'm not sure if replacing Confluence is an aim of the Outline developers, but it feels this kind of task related management options could be better served from workflow engines like @taigaio even GitLab, which has a quite sophisticated tasks engine. Alternatively, you could also try building something like this out of @discourse and a blend of theme components + plugins.

**Reply by almereyda** [reply]
Another alternative to Confluence, which seems to be similarily burdened with many functionalities at once, is also XWiki.

**Comment by githubkusi** [comment]
Same as https://github.com/outline/outline/discussions/4241

**Comment by dizummd21** [comment]
Gitlab OnPrem Issues linking would be nice.
